app/Models/Channel.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Channel extends Model
+{
+    use HasFactory;
```

Comment:
You can use `Sluggable` train from *blaze* to auto handle creating slugs for the channel.